Code-the-Dream-School/dd-prac-team4-front
# Loader implenented

Loader component successfully created.
This is functional styled 'Loader' component that displays a spinner with optional size variations based on the given class name.

Testing Instructions:

Install the necessary dependencies by running npm install in the project directory.
Signup to the application 
Navigate to Home page http://localhost:3000/ 
Verify that the Home page displays a loader
